xbmc/cores/RetroPlayer/streams/RetroPlayerAudio.cpp

```diff
@@ -118,8 +122,19 @@ void CRetroPlayerAudio::AddStreamData(const StreamPacket &packet)
   {
     if (m_pAudioStream)
     {
+      const double delay = m_pAudioStream->GetDelay();
+
       const size_t frameSize = m_pAudioStream->GetChannelCount() * (CAEUtil::DataFormatToBits(m_pAudioStream->GetDataFormat()) >> 3);
-      m_pAudioStream->AddData(&audioPacket.data, 0, static_cast<unsigned int>(audioPacket.size / frameSize));
+
+      const unsigned int frameCount = static_cast<unsigned int>(audioPacket.size / frameSize);
+
+      if (delay > 0.3)
```

Comment:
I missed this when reviewing, can you move the magic number to the top of the file right after `using namespace RETRO;`? Avoid using a #define